src/multi_vector_simulator/F1_plotting.py

```diff
@@ -855,52 +856,70 @@ def create_plotly_flow_fig(
         figure object
     """
 
-    fig = go.Figure()
-    styling_dict = get_fig_style_dict()
-    styling_dict["gridwidth"] = 1.0
+    def create_plot(
+        flow_list, filename=file_name, y_legend=y_legend, plot_title=plot_title
+    ):
+        fig = go.Figure()
+        styling_dict = get_fig_style_dict()
+        styling_dict["gridwidth"] = 1.0
+
+        for i, asset in enumerate(flow_list):
+            fig.add_trace(
```

Comment:
How does that filter out the column with the SOC? I mean, it does, but I do not understand how:
![Electricity_bus_power](https://user-images.githubusercontent.com/44204527/99689216-03f0db00-2a87-11eb-9d83-8ea5d944b09c.png)

Reply:
It's removed [here](https://github.com/rl-institut/multi-vector-simulator/blob/ab28f83a1ca89e365c946d756564acf1f484949d/src/multi_vector_simulator/F1_plotting.py#L919-L920). I'll add more comments to make that clear before the block "`if any(SOC in item for item in assets_list):`" starts (the comment was hidden in the block).